packages/fxa-settings/README.md

```diff
@@ -11,7 +11,26 @@ You can import React components into this project. This is currently restricted
 
 ```javascript
 // e.g. assuming the component HelloWorld exists
-import HelloWorld from "@fxa-components/HelloWorld";
+import HelloWorld from '@fxa-components/HelloWorld';
+```
+
+**Working with SVGs**
+
+Create React App allows us to use SVGs in a variety of ways, right out of the box.
+
+```javascript
+// As an image source:
+import Logo from './logo.svg';
+const LogoImage = () => <img src={Logo} alt="Logo" />;
+
+// As a background-image (inline style)
+import Logo from './logo.svg';
+const LogoImage = () => <div style={{ backgroundImage: `url(${Logo})` }}></div>;
+
+// As a background-image (external style)
+// Just reference it in CSS, the loader will find it
+// .logo { background-image: url('logo.svg'); }
+const LogoImage = () => <div class="logo"></div>;
```

Comment:
Oh, one more comment here, we should probably have `role="img"` and `aria-label="logo"` on line 33 just for best practice. We could have a note on the first option being the preferred method too, the second option starts to mix CSS-in-JS but we could do that if we needed to for some reason.